lib/src/localization/strings_it.i18n.json

```
@@ -14,16 +13,26 @@
     "objects": "Oggetti",
     "symbols": "Simboli",
     "flags": "Bandiere"
+    "custom": "Personalizzato"
   },
-
   "settings": {
     "title": "Impostazioni",
+    "closeToTray": "Vicino alla barra delle applicazioni",
     "theme": "Tema",
     "systemTheme": "Tema di sistema",
     "lightTheme": "Tema Chiaro",
     "darkTheme": "Tema Scuro",
+    "showSystemTray": "Mostra l'icona nella barra delle applicazioni",
     "exitAfterCopy": "Esci dopo aver copiato"
```

This'll probably fail test for missing trailing comma as well

```suggestion
    "exitAfterCopy": "Esci dopo aver copiato",
```